CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix a reconciliation bug on CAPI MC that were reconciled twice.
+- Ensure the remote write endpoint configuration is enabled for MCs as well
```

Comment:
I see we are fixing 2 different problems here I would prefer to have those in 2 separate PRs. It ease up reviews and makes reverting changes in case of problem easier.